Source/WarpX.H

```diff
@@ -253,6 +253,8 @@ public:
     int mag_LLG_exchange_coupling = 0;
     // turn off the anisotropy coupling term H_anisotropy in H_eff for the LLG updates
     int mag_LLG_anisotropy_coupling = 0;
+    // The anisotropy_axis for the anisotropy coupling term H_anisotropy in H_eff
+    amrex::Vector<amrex::Real> mag_LLG_anisotropy_axis;
```

Comment:
```suggestion
```

Reply:
To ensure all macroscopic properties are in one place, this should go in MacroscopicProperties.H as a GpuArray.
You can use a temporary vector to ParmParse the values in MacroscopicProperties::ReadParameters()